website/src/pages/blog/[...slug].astro

```diff
@@ -19,11 +20,36 @@ const { entry } = Astro.props;
 const { Content } = await entry.render();
 
 const post = entry.data;
+
+const processedImage = await getImage({ src: post.socialImage });
+const socialImageUrl = new URL(
+  processedImage.src,
+  Astro.site ?? "http://localhost:4321/"
```

Comment:
Can the right hand side be `Astro.url.origin`? It should be more future proof, especially if you run multiple astro dev servers

Reply:
Oh nice, forgot we had this property 😅 Just for context og images need to be an absolute link because of some weird limitations in how these work, so I've wrapped it into a template literal with a trailing slash.